library/formatting.inc.php

```diff
@@ -216,3 +216,24 @@ function oeFormatAge($dobYMD, $nowYMD = '', $format = 0)
 
     return $age;
 }
+
+
+/**
+ * Returns the complete formatted stae string according the global date format
+ * @param $timestamp
+ * @return bool|string
+ */
+function oeFormatLongDate($timestamp)
+{
+    if(!$timestamp){
+        $timestamp = date('Y-m-d H:i');
+    }
+    // assume input is yyyy-mm-dd
+    if ($GLOBALS['date_display_format'] == 1) {      // mm/dd/yyyy, note year is added below
+        $newDate = date('m-d-Y H:i',$timestamp);
```

Comment:
the timestamp (unix timestamp) is a unit that counts the seconds since 1970.
So 1502173654 = Tuesday 8th August 2017 06:27:34 AM

Its a very known and easy way to handle dates.